src/modules/fw_pos_control_l1/FixedwingPositionControl.hpp

```diff
@@ -362,74 +422,193 @@ class FixedwingPositionControl final : public ModuleBase<FixedwingPositionContro
+	 */
+	void control_manual_position(const float control_interval, const Vector2d &curr_pos, const Vector2f &ground_speed);
+
+	float get_tecs_pitch();
+	float get_tecs_thrust();
+
+	float get_manual_airspeed_setpoint();
 
 	/**
-	 * @brief Returns an indicated airspeed setpoint for auto modes.
+	 * @brief Returns an calibrated airspeed setpoint for auto modes.
 	 *
 	 * Adjusts the setpoint for wind, accelerated stall, and slew rates.
 	 *
 	 * @param control_interval Time since the last position control update [s]
 	 * @param pos_sp_cru_airspeed The commanded cruise airspeed from the position setpoint [s]
 	 * @param ground_speed Vehicle ground velocity vector [m/s]
-	 * @return Indicated airspeed setpoint [m/s]
+	 * @return Calibrated airspeed setpoint [m/s]
 	 */
 	float get_auto_airspeed_setpoint(const float control_interval, const float pos_sp_cru_airspeed,
 					 const Vector2f &ground_speed);
 
-	void		reset_takeoff_state(bool force = false);
-	void		reset_landing_state();
-	bool		using_npfg_with_wind_estimate() const;
-	Vector2f 	get_nav_speed_2d(const Vector2f &ground_speed);
-	void		set_control_mode_current(const hrt_abstime &now, bool pos_sp_curr_valid);
+	void reset_takeoff_state(bool force = false);
+	void reset_landing_state();
+
+	bool using_npfg_with_wind_estimate() const;
+
+	/**
+	 * @brief Returns the velocity vector to be input in the lateral-directional guidance laws.
+	 *
+	 * Replaces the ground velocity vector with an air velocity vector depending on the wind condition and whether
+	 * NPFG or L1 are being used for horizontal position control.
+	 *
+	 * @param ground_speed Vehicle ground velocity vector [m/s]
+	 * @return Velocity vector to control with lateral-directional guidance [m/s]
+	 */
+	Vector2f get_nav_speed_2d(const Vector2f &ground_speed);
+
+	/**
+	 * @brief Decides which control mode to execute.
+	 *
+	 * May also change the position setpoint type depending on the desired behavior.
+	 *
+	 * @param now Current system time [us]
+	 * @param pos_sp_curr_valid True if the current position setpoint is valid
+	 */
+	void set_control_mode_current(const hrt_abstime &now, bool pos_sp_curr_valid);
 
 	void publishOrbitStatus(const position_setpoint_s pos_sp);
 
-	SlewRate<float> _slew_rate_airspeed;
+	SlewRate<float> _airspeed_slew_rate_controller;
 
-	/*
-	 * Call TECS : a wrapper function to call the TECS implementation
+	/**
+	 * @brief A wrapper function to call the TECS implementation
+	 *
+	 * @param control_interval Time since the last position control update [s]
+	 * @param alt_sp Altitude setpoint, AMSL [m]
+	 * @param airspeed_sp Calibrated airspeed setpoint [m/s]
+	 * @param pitch_min_rad Nominal pitch angle command minimum [rad]
+	 * @param pitch_max_rad Nominal pitch angle command maximum [rad]
+	 * @param throttle_min Minimum throttle command [0,1]
+	 * @param throttle_max Maximum throttle command [0,1]
+	 * @param throttle_cruise Cruise throttle command [0,1]
```

Comment:
It's not really a command but rather a TECS setting no? So something like "Configured throttle for level flight"